docs/usage/configuration/config_documentation.md

```diff
@@ -474,7 +489,7 @@ Valid resource names are:
 
 * `media`: the media API (/_matrix/media).
 
-* `metrics`: the metrics interface. See [here](../../metrics-howto.md).
+* `metrics`: the metrics interface. See [here](../../metrics-howto.md). (Not usable on a UNIX Socket `listener`)
```

Comment:
> (Not usable on a UNIX Socket `listener`)

Why?

Reply:
The short story is that it's not servable in it's present form. The long story is in the other comment https://github.com/matrix-org/synapse/pull/15708#discussion_r1227922083

Reply:
```suggestion
* `metrics`: the metrics interface. See [here](../../metrics-howto.md). (Not compatible with Unix sockets)
```

Reply:
Can we throw a config error if someone uses a Unix socket with `metrics`?